wavesexchange_topic/src/lib.rs

```diff
@@ -942,6 +1125,13 @@ impl TopicData {
         }
     }
 
+    pub fn as_pairs(&self) -> Option<&Vec<ExchangePairs>> {
+        match self {
+            TopicData::Pairs(pairs) => Some(&pairs),
+            _ => None,
+        }
+    }
+
```

Comment:
This comment goes below, to the `test_eq_and_hash()` unit test.
Pls add pairs URL to that test also for completeness.